Library/Homebrew/os/mac/xcode.rb

```diff
@@ -15,7 +15,7 @@ def latest_version
         when "10.8"  then "5.1.1"
         when "10.9"  then "6.2"
         when "10.10" then "7.2.1"
-        when "10.11" then "7.3.1"
+        when "10.11" then "8.0"
```

Comment:
note that this may be controversial to enforce on 10.11, but it doesn't bother me